update-keys.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh -ex
 
-curl -fsSLo- --compressed https://github.com/nodejs/node/raw/master/README.md | awk '/^gpg --keyserver pool.sks-keyservers.net --recv-keys/ {print $NF}' > keys/node.keys
+curl -fsSLo- --compressed https://github.com/nodejs/node/raw/master/README.md | awk '/^gpg --keyserver hkps://keys.openpgp.org --recv-keys/ {print $NF}' > keys/node.keys
```

Comment:
This is a regex, so I think you wanted to leave this unless/until that README gets updated right?  (alternatively, you could relax it with something like `[^ ]+`)

Reply:
Ah, good point

Reply:
I opened https://github.com/nodejs/node/pull/39227 to updated the README, but this script isn't used often, so i'm not too worried if it temporarily breaks